INN/umbrella-ipbs
# IPBS member stations widget and styling

## Changes

This pull request makes the following changes:

- Adds the IPBS Member Stations widget along with relevant styling for it

Desktop:
![screencapture-ipbs-test-2020-01-29-16_07_02](https://user-images.githubusercontent.com/18353636/73397498-a9954c80-42b1-11ea-8cc2-c8169de34926.png)

Mobile:
![screencapture-ipbs-test-2020-01-29-16_07_24](https://user-images.githubusercontent.com/18353636/73397499-a9954c80-42b1-11ea-812e-d947c75e5437.png)

## Why



For #58

## Testing/Questions

Features that this PR affects:

- IPBS member stations widget


Questions that need to be answered before merging:

- [ ] For the mobile version (and desktop, too, kinda), instead of being able to horizontally center the station content, I had to keep it left aligned so that the arrow transition still works as expected. Is that ok?

Steps to test this PR:

1. Add the IPBS Member Stations widget to the homepage bottom widget area
2. View it on desktop and mobile and make sure nothing funky happens with it
